server/www/templates/common/filter-system/filter-searchbar.html

```diff
@@ -1,9 +1,9 @@
 <div class="searchbar-input" dropdown auto-close="outsideClick">
     <!-- Search input -->
     <div class="form-field search">
-        <input name="q" type="search" 
-        placeholder="{{ 'toolbar.searchbar.search_entity' | translate:{entity: placeholderText} }}" 
-        ng-model="q">
+        <input name="q" type="search"
+        placeholder="{{ 'toolbar.searchbar.search_entity' | translate:{entity: placeholderText} }}"
+        ng-model="filters.q">
```

Comment:
The way I had done this in users was that in the filter-users applyFilters() it copies the value of the q into filters.q. Wiring it directly to filters.q caused the $watch to fire the moment filters.q was updated so before the user hit enter/applyFilters(). This is because I was using the deep $watch setting. 
Without deep watch the $watch wasn't being fired because it $watch consider the object not to have been changed.
I'm not sure what the best work around for this would be but I would much prefer this to be filters.q and to drop having to ask the form for the value of q